frontend/src/__tests__/cypress/cypress/pages/servingRuntimes.ts

```diff
@@ -123,6 +123,22 @@ class ServingRuntimes {
   getRowById(id: string) {
     return new ServingRuntimeRow(id);
   }
+
+  editModel() {
+    return cy.findByLabelText('Kebab toggle');
+  }
+
+  deleteModel() {
+    return cy.findByRole('menuitem', { name: 'Delete' });
+  }
+
+  deleteModal() {
+    return cy.get('[data-testid="delete-modal-input"]');
+  }
+
+  deleteModelServingButton() {
```

Comment:
Based on the guidelines in `docs/testing.md` for naming functions, they should start with `find...`

Reply:
agree, it just returns the locator, but does not click on it yet.